packages/run-it/src/RunIt.tsx

```diff
@@ -162,6 +163,11 @@ export const RunIt: FC<RunItProps> = ({
     }
   }, [hasConfig, isExtension, needsAuth, sdk])
 
+  const handleConfig = (e: BaseSyntheticEvent) => {
+    e.preventDefault()
```

Comment:
is preventDefault needed?

Reply:
It's need for form submit, I dont think its needed for button click (i believe this is what it associated with).

For handleSubmit I would call preventDefault in the RequestForm component.

Again, change not needed for this PR.

Reply:
Definitely worth testing these overrides to see if they're needed. Is it just to prevent the "submit" action? I'll find out tomorrow.